plugins/sonarqube/src/components/SonarQubeCard/SonarQubeCard.tsx

```diff
@@ -232,7 +232,9 @@ export const SonarQubeCard = ({
                 link={value.getComponentMeasuresUrl('COVERAGE')}
                 title="Coverage"
                 leftSlot={<Percentage value={value.metrics.coverage} />}
-                rightSlot={<Value value={`${value.metrics.coverage}%`} />}
+                rightSlot={
+                  <Value value={`${value.metrics.coverage || '-'}%`} />
```

Comment:
Not `-%` tho :p

Reply:
And maybe, just to be consistent to the one above, use `—` instead of `-`?